### PR TITLE
Add mentoring notes for the elisp two-fer exercise

### PR DESCRIPTION
**Pull request by indiebrain**
Summary
-------

I find the mentoring notes invaluable in providing consistent feedback
to students however the elisp track has none. It's by no means meant
as an authoritative "The exercise MUST be done this way", but rather
distills some of the most repeated points I've delivered in the past
few weeks #contributions-welcome

### CODE REVIEW

File: tracks/elisp/exercises/two-fer/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "One for NAME, one for me."
+  (format "One for %s, one for me." (or name "you")))
+
+(provide 'two-fer)
+;;; two-fer.el ends here
+```
+
+### Common suggestions
+
+* Suggest the use of a single call to the `format` function - https://www.gnu.org/software/emacs/manual/html_node/elisp/Formatting-Strings.html
```
Comment (SleeplessByte) [inline tracks/elisp/exercises/two-fer/mentoring.md]:
```suggestion
* Suggest the use of a single call to the `format` function - [https://www.gnu.org/software/emacs/manual/html_node/elisp/Formatting-Strings.html](https://www.gnu.org/software/emacs/manual/html_node/elisp/Formatting-Strings.html)
```

File: tracks/elisp/exercises/two-fer/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* Explain that the `format` function takes a format string and a list of objects to substitute into the format string, and that those objects can - themselves - be expressions. For example:
```
Comment (SleeplessByte) [inline tracks/elisp/exercises/two-fer/mentoring.md]:
```suggestion
* Explain that the `format` function takes a format string and a list of objects to substitute into the format string, and that those objects can _themselves_ be expressions. For example:
```

File: tracks/elisp/exercises/two-fer/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+```elisp
+(format "1 for you, %s for me" (+ 1 1)) ;;; "1 for you, 2 for me"
+```
+* Suggest the use of the `or` special form over complex conditionals like `(if name name "you")` - https://www.gnu.org/software/emacs/manual/html_node/elisp/Combining-Conditions.html#Combining-Conditions
```
Comment (SleeplessByte) [inline tracks/elisp/exercises/two-fer/mentoring.md]:
```suggestion
* Suggest the use of the `or` special form over complex conditionals like `(if name name "you")` - [https://www.gnu.org/software/emacs/manual/html_node/elisp/Combining-Conditions.html#Combining-Conditions](https://www.gnu.org/software/emacs/manual/html_node/elisp/Combining-Conditions.html#Combining-Conditions)
```

File: tracks/elisp/exercises/two-fer/mentoring.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+### Talking points
+
+* The `defun` macro takes several arguments, one of which is a documentation string of the function. In addition to giving more context to the reader of the source code Emacs' help system can display this documentation to consumers of your function. The documentation string parameter of the `defun` macro _is_ optional, but highly encouraged by the community. https://www.gnu.org/software/emacs/manual/html_node/eintr/defun.html
```
Comment (SleeplessByte) [inline tracks/elisp/exercises/two-fer/mentoring.md]:
```suggestion
* The `defun` macro takes several arguments, one of which is a documentation string of the function. In addition to giving more context to the reader of the source code Emacs' help system can display this documentation to consumers of your function. The documentation string parameter of the `defun` macro _is_ optional, but highly encouraged by the community. [https://www.gnu.org/software/emacs/manual/html_node/eintr/defun.html](https://www.gnu.org/software/emacs/manual/html_node/eintr/defun.html)
```

**Comment by indiebrain** [comment]
Many thanks for the review and suggestions @SleeplessByte !

**Comment by NobbZ** [comment]
There are no members in @exercism/website-copy-elisp.

Could therefore one of @exercism/website-copy-common-lisp, @exercism/website-copy-scheme, @exercism/website-copy-clojure, or @exercism/website-copy-lfe confirm if these mentoring notes seem reasonable from your point of view?

I think it is safe to merge in 48 hours if no one vetos.

**Comment by SleeplessByte** [comment]
Thank you for the review @verdammelt

**Pull request merged**